ci-operator/step-registry/cucushift/upgrade/mirror-images/by-oc-adm/cucushift-upgrade-mirror-images-by-oc-adm-commands.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function mirror_image(){
+    if (( $# < 1 )); then
+        echo >&2 "Expect a release image passed, but found 0" && return 1
+    fi
+
+    target_image="${1}"
+    mirror_release_image="${MIRROR_REGISTRY_HOST}/${target_image#*/}"
+    mirror_release_image_repo="${mirror_release_image%:*}"
+    mirror_release_image_repo="${mirror_release_image_repo%@sha256*}"
+
+    target_version=$(oc adm release info "${target_image}" --output=json | jq .metadata.version)
+   
+    echo "Mirroring ${target_version} (${target_image}) to ${mirror_release_image_repo}"
+
+    oc adm release mirror -a "${new_pull_secret}" --insecure=true \
+        --from="${target_image}" \
+        --to="${mirror_release_image_repo}" \
```

Comment:
Miss "--to-release-image" or new change? I see it's stilled used in `ci-operator/step-registry/mirror-images/by-oc-adm/mirror-images-by-oc-adm-commands.sh`.

Reply:
Hmm, I got the cli from the doc https://docs.openshift.com/container-platform/4.10/updating/updating-restricted-network-cluster.html#update-mirror-repository_updating-restricted-network-cluster.

Reply:
Per my memory, if no `--to-release-image` option, will no payload image will be created, no idea if some new change recently, but to be consistent, suggest to always using `--to-release-image` option there.

Reply:
OK. If using `--to-release-image` is the best practice, should we update the doc? My concern is if we test A but tell customers to do B, seems it doesn't make much sense.

Reply:
If it works as expected, that's ok for me to leave it as current status. And this can help cover the doc's test.

Reply:
I saw doc is using `--to-release-image` option in https://docs.openshift.com/container-platform/4.10/installing/disconnected_install/installing-mirroring-installation-images.html

Reply:
yeah, the cli in installation doc differs from upgrade doc. Per the oc help:
--to-release-image='': Specify an alternate locations for the release image instead as tag 'release' in --to.

It seems that if we specify different location for --to-release-image from --to, then it makes sense. Otherwise, seems no need to use --to-release-image.

Reply:
okay. get it.